org.virt_manager.virt-manager.yaml

```diff
@@ -79,6 +79,18 @@ modules:
       - shared-modules/intltool/intltool-0.51.json
       - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
 
+      - name: xorriso
+        buildsystem: autotools
+        sources:
+          - type: archive
+            url: https://www.gnu.org/software/xorriso/xorriso-1.5.6.pl02.tar.gz
+            sha256: 786f9f5df9865cc5b0c1fecee3d2c0f5e04cab8c9a859bd1c9c7ccd4964fdae1
+            x-checker-data:
+              type: html
```

Comment:
Can we use [anitya](https://github.com/flathub-infra/flatpak-external-data-checker#anitya-release-monitoring-checker) here with: https://release-monitoring.org/project/13270/ ?